src/scripts/datadog-docs.js

```diff
@@ -79,36 +80,22 @@ $(document).ready(function () {
         }
     }
 
-    codeTabs();
-
-    // ------------- TODO: move TOC js back to own file when webpack migration complete and can import js modules
-
     updateTOC();
     buildTOCMap();
     onScroll();
 
-    // TODO: move integrations code to own file after webpack update
-
     if (document.body.classList.value.includes('security_platform')) {
         initializeSecurityRules();
     }
 
     if (document.body.classList.value.includes('integrations')) {
         initializeIntegrations();
     }
-});
-
 
-
-function getParameterByName(name, url) {
```

Comment:
we already have a helper function `getQueryParameterByName` so this is no longer necessary